RocketChat/Rocket.Chat
# fix: Embedded layout stops working after navigation occurs

## Proposed changes (including videos or screenshots)





## Issue(s)


## Steps to test or reproduce

Go to admin -> layout -> User Interface and enable "Show top navbar in embedded layout"
After this, enter any channel and add `?layout=embedded` to the end of the site url in your browser (should look something like this: `https://localhost:3000/channel/GENERAL?layout=embedded`)
Open any contextual bar (thread list, room info, etc). It loses the embedded layout.
## Further comments
